src/FluentHub.Octokit/Queries/Users/NotificationQueries.cs

```diff
@@ -1,25 +1,227 @@
-﻿namespace FluentHub.Octokit.Queries.Users
+﻿using GraphQL;
+
+namespace FluentHub.Octokit.Queries.Users
 {
     public class NotificationQueries
     {
-        public async Task<List<Notification>> GetAllAsync(OctokitOriginal.NotificationsRequest request = null, OctokitOriginal.ApiOptions options = null)
+        public async Task<List<Notification>> GetAllAsync(OctokitV3.NotificationsRequest request = null, OctokitV3.ApiOptions options = null)
         {
             var response = await App.Client.Activity.Notifications.GetAllForCurrent(request, options);
 
             Wrappers.NotificationWrapper wrapper = new();
-            var notifications = await wrapper.WrapAsync(response);
+            var notifications = wrapper.WrapAsync(response);
+
+            var fragments = GetGetheredRepositoryFragment(notifications);
+
+            var request2 = new GraphQLRequest
+            {
+                Query = @$"query {{ {fragments} }}",
+            };
+
+            var response2 = await App.GraphQLHttpClient.SendQueryAsync<NotificationSubjectDetails>(request2);
+            List<Repository> zippedData = new()
+            {
+                response2.Data.repo0,
```

Comment:
This looks very weird... Do other ways exist?

Reply:
I think so. No, there's no way.

Reply:
Tomorrow I will look, what we can do with it, because its ridiculous and not serious 😅